### PR TITLE
fix: typo

### PR DESCRIPTION
**Pull request by mrfoe7**

### CODE REVIEW

File: redis_broadcast.go [hunk 1]
```diff
@@ -375,10 +373,12 @@ func (bc *redisBroadcast) Send(room, event string, args ...interface{}) {
 	defer bc.lock.RUnlock()
 
 	connections, ok := bc.rooms[room]
-	if ok {
-		for _, connection := range connections {
-			connection.Emit(event, args...)
-		}
+	if !ok {
+		return
```
Comment (erkie) [inline redis_broadcast.go:377]: This changes the method. Previously `bc.publishMessage(room, event, args...)` was always called. Was that a bug or intentional?

File: namespace_conn.go [hunk 2]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/googollee/go-socket.io/parser"
 )
 
-// Namespace
+// Namespace is interface describe namespace functionality
```
Comment (erkie) [inline namespace_conn.go:11]: Not really sure what this means?

Should it be something like?

```go
// Namespace describes a socket.io room
```

or similar?

**Review by erkie — CHANGES_REQUESTED** [review]
Great changes. Just 2 questions that could be improved :)

File: redis_broadcast.go (hunk 1 above)
Comment (mrfoe7) [reply redis_broadcast.go:377]: It is refactoring and not more

File: namespace_conn.go (hunk 2 above)
Comment (mrfoe7) [reply namespace_conn.go]: Nice one. I changed

File: namespace_conn.go [hunk 3]
```diff
@@ -8,7 +8,7 @@ import (
 	"github.com/googollee/go-socket.io/parser"
 )
 
-// Namespace
+// Namespace describes a socket.io room.
```
Comment (googollee) [inline namespace_conn.go:11]: `Namespace describes a communication channel that allows you to split the logic of your application over a single shared connection.`

A namespace is not a room, see https://socket.io/docs/v2/namespaces/.

File: redis_broadcast.go (hunk 1 above)
Comment (googollee) [reply redis_broadcast.go:377]: This refactor changes the logic. If there is no room, it won't call publishMessage(), and other nodes won't receive the message.

**Review by googollee — CHANGES_REQUESTED** [review]

File: redis_broadcast.go (hunk 1 above)
Comment (mrfoe7) [reply redis_broadcast.go]: Sorry, guys. It is my fault. Erik, you right. Rollbacked

**Comment by erkie** [comment]
Nice work! Great to see this type of cleanup

**Pull request merged**